prody/dynamics/anm.py

```diff
@@ -302,7 +302,7 @@ def calcModes(self, n_modes=20, zeros=False, turbo=True):
 
         if self._hessian is None:
             raise ValueError('Hessian matrix is not built or set')
-        if str(n_modes) is 'all':
+        if str(n_modes).lower == 'all':
             n_modes = None
```

Comment:
This one is missing brackets after lower. Yes, ‘is’ is bad for string comparison, especially when you have encode and binary strings to compare

Reply:
ok